### PR TITLE
Create Bot

### PR DESCRIPTION
**Pull request by Hoai12**

### CODE REVIEW

**Comment by skamecka7** [comment]
> _Nie podano opisu._

[bugreport-dandelion_eea-QP1A.190711.020-2022-07-23-16-38-27-dumpstate_log-1518.txt](https://github.com/github/haikus-for-codespaces/files/9176868/bugreport-dandelion_eea-QP1A.190711.020-2022-07-23-16-38-27-dumpstate_log-1518.txt)

**Review by mwildrick4389 — APPROVED** [review]

**Comment by Benny6731** [comment]
hey

File: Bot [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (Oky66) [inline Bot:1]: $ heroku login

Comment (Hodocharles) [reply Bot:1]: Hi

**Pull request closed without merging**